### PR TITLE
Deprecate 16px flag

### LINKED ISSUE #2836: Deprecate 16px flag
## Background

OGS maintains [a fork of world-flags-sprite](https://github.com/online-go/world-flags-sprite), which it uses to display national flags around the site.  There are two supported resolutions, 16px and 32px. Maintaining two sets of images is a pain, and there is a lingering bug (#2043) in the 16px flag set.

## Suggested Changes

- Re-implement [`<Flag country=".." big=false>`](https://github.com/online-go/online-go.com/blob/2e8bbd16e116b5b7318f312a91bf4dd1388a4d58/src/components/Flag/Flag.tsx#L28) to always use "f32", then scale the small flag accordingly.
    - Check that small flags are showing up with the same layout as they did before.  One example is the [Ladders page](https://online-go.com/ladder/313).
- [Cleanup] remove references to the 16px class from this repo
    - https://github.com/online-go/online-go.com/blob/2e8bbd16e116b5b7318f312a91bf4dd1388a4d58/src/global_styl/flags16.styl#L20 
    - https://github.com/online-go/online-go.com/blob/2e8bbd16e116b5b7318f312a91bf4dd1388a4d58/src/components/Flag/Flag.styl#L19
    - https://github.com/online-go/online-go.com/blob/2e8bbd16e116b5b7318f312a91bf4dd1388a4d58/src/components/Player/PlayerDetails.styl#L40
    - ... more?
- [Cleanup] Remove the 16px spritesheet from the [flags repo](https://github.com/online-go/world-flags-sprite)

## Related PRs

- https://github.com/online-go/online-go.com/pull/2042
- https://github.com/online-go/world-flags-sprite/pull/4

### PR DESCRIPTION
**Pull request by tylerbarker**
Fixes #2836 

# Overview

Hey folks, new here (to the game and OGS) 👋🏼 

Hope you don't mind me picking it up - I didn't call it out on the issue - but I decided to have a go deprecating the 16px flags since it appeared to still be up for grabs. Thanks @benjaminpjones for the detailed outline.

The changes are more or less as specified, however I had to wrap the Flag in a new parent with `inline-flex` applied, as there were a number of cases around the site using `inline-block` that had to be overridden in order to have the scale transform apply.

You'll note PlayerDetails also required a little tweaking beyond what was already specified previously in the Stylus file under `.f16` - that wasn't positioning correctly any longer.

The last thing was just removing the height/width specified in ChatChannelList styles as that was squashing the downscaled flags.

I've taken a heap of screenshots across the app for peace of mind - tested in Chrome & Safari.

Let me know your thoughts.

## Next Up

Assuming all's well, I'll move on to removing the 16px sprite set from the `world-flags-sprite` fork once this is merged.

## Screenshots
prSettingsShot
prFlagDropdownShot
prChatChannelListShot
prChatListShot
prProfileShot
prPlayerDetailsShot
prGoResourcesShot
prTeamShot
prGameShot
prTeamShot2

### CODE REVIEW

File: src/components/Flag/Flag.styl [hunk 1]
```diff
@@ -16,10 +16,13 @@
...
 
-.f16, .f32 {
+.f32 {
     .flag {
         vertical-align: middle;
         margin-bottom: 0.1em;
     }
+    &.flag-small {
+      transform: scale(0.5, 0.5);
+      margin: -8px -8px -8px -8px;
```
Comment (tylerbarker) [inline src/components/Flag/Flag.styl:26]: I don't love negative margins, but without this `f32` insisted on rendering a 32px container around the downscaled image - open to other ideas if you have them but this appears to work and maintain the layouts as they are now.

Comment (anoek) [reply src/components/Flag/Flag.styl:26]: Yeah it seemed a little weird at first glance, but I messed around a bit with some alternative approaches and they seemed to have various faults, and this seems to work well. Thanks for the solution @tylerbarker !

Tagging @benjaminpjones as he's stronger with some of this stuff than I for some additional sanity checking.

Comment (anoek) [reply src/components/Flag/Flag.styl:26]: Thanks for the followup @benjaminpjones !

**Comment by anoek** [comment]
Thanks @tylerbarker !

**Pull request merged**